runtime/src/bank_forks.rs

```diff
@@ -67,6 +73,10 @@ impl BankForks {
         self.banks.clone()
     }
 
+    pub fn in_vote_only_mode(&self) -> Arc<AtomicBool> {
```

Comment:
```suggestion
    pub fn get_vote_only_mode_signal(&self) -> Arc<AtomicBool> {
```

original name made me think I'm getting a `bool`